styles/styles.css

```
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css?family=Lora|Manjari&display=swap');
+
+body {
+  font-family: "Manjari";
+  border: solid whitesmoke 10px;
+}
+
+.navbar {
+  position: fixed;
+  padding-left: 58px;
+}
+
+nav ul {
+  list-style: none;
+  text-align: right;
+  font-weight: bold;
+  display: grid;
+  grid-template-columns: 1fr 8.5fr 8fr;
+  position: relative;
+  font-size: 20px;
+}
+
+h2 {
+  text-align: center;
+  font-size: 30px;
+  font-family: "Lora";
+}
+
+#title {
+  display: flex;
+  font-size: 25px;
+  align-items: center;
+  flex-direction: column;
+  justify-content: space-between;
+  border-bottom-style: solid;
+  border-bottom-color: gray;
+  border-bottom-width: thin;
+  padding-top: 8rem;
+  padding-bottom: 3rem;
+  background-image: url("https://images.pexels.com/photos/205316/pexels-photo-205316.png?cs=srgb&dl=apple-clean-containers-205316.jpg&fm=jpg");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
```

The parallax above was really cool! I think for clarity I would suggest adding another 'id' to the title section and separating out the parallax-specific code from the title code.

```suggestion
#parallax {
  background-image: url("https://images.pexels.com/photos/205316/pexels-photo-205316.png?cs=srgb&dl=apple-clean-containers-205316.jpg&fm=jpg");
  background-repeat: no-repeat;
  background-position: center center;
  background-attachment: fixed;
  background-size: cover;
  }
```